stories/filters.js

```diff
@@ -108,15 +107,15 @@ const fileSectionsWithTooltips = [
 ];
 
 const tabs = [
-  <FilterList key={0} sections={projectSections} />,
-  <FilterList key={1} sections={subjectSections} />,
-  <FilterList key={2} sections={fileSections} />,
+  <FilterList key={0} sections={projectSections} tierAccessLimit={1000} />,
+  <FilterList key={1} sections={subjectSections} tierAccessLimit={1000} />,
+  <FilterList key={2} sections={fileSections} tierAccessLimit={1000} />,
```

Comment:
What's the reason for this change? Is the default tierAccessLimit value messing up the story view? Just curious

Reply:
The `tierAccessLimit` should come from guppy but since there is no guppy so it needs to be put in manually. Otherwise the disabled filter in storybook will only have a "⬇️"

Nothing major, just some cosmetic issues